bridge-impl/src/main/java/com/liferay/faces/bridge/application/internal/ResourceHandlerInnerImpl.java

```diff
@@ -110,4 +110,9 @@ else if (!resourceValidator.isValidLibraryName(libraryName)) {
 			super.handleResourceRequest(facesContext);
 		}
 	}
+
+	@Override
+	public boolean isResourceURL(String url) {
+		return (url != null) && url.contains("javax.faces.resource=");
```

Comment:
@ngriffin7a, I implemented this method so that it would only return `true` for portlet faces resourceURLs (*not* webapp faces resourceURLs which contain `"javax.faces.resource"` without the equals sign `"="`), please let me know if that is incorrect.